infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/IClientServerRouter.java

```diff
@@ -0,0 +1,75 @@
+package org.corfudb.infrastructure.logreplication.transport;
+
+import org.corfudb.runtime.LogReplication;
+import org.corfudb.runtime.proto.service.CorfuMessage;
+import javax.annotation.Nonnull;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+
+public interface IClientServerRouter {
```

Comment:
can you add javadocs to these new classes?

Reply:
+1